index.md

```diff
@@ -9,10 +9,10 @@ headerButtonUrl: "/what-is-scala/"
 
 # Links of the Download / API Docs sections
 gettingStarted:
-  mainTitle: "Download"
-  mainUrl: "/download"
-  subtitle: "Getting Started"
-  subtitleLink: "https://docs.scala-lang.org/getting-started.html"
+  mainTitle: "Getting Started"
+  mainUrl: "https://docs.scala-lang.org/getting-started.html"
+  subtitle: "Download"
+  subtitleLink: "/download"
```

Comment:
Hum, the new Download page is more beautiful/more engaging than the Getting Started page. IMO we should leave it as the bigger button, so keep this unchanged.

Reply:
It's in fact more beautiful but not "beginners' oriented". It contains release notes, Scala archives.
Putting the getting started first, shows that we give more importance to the learning process. 
Maybe we can have the same style of CSS in the getting started, and remove duplicated information from the download page?

Reply:
> It's in fact more beautiful but not "beginners' oriented". It contains release notes, Scala archives.
Putting the getting started first, shows that we give more importance to the learning process.

I guess that makes sense. Let's keep it as is, then.